packages/bpk-docs/src/components/neo/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -59,9 +63,50 @@ class DocsPageWrapper extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = { selected: this.props.nativeSubpage ? 'native' : 'web' };
+
+    this.state = {
+      selected: this.getSelectedSubpage(),
+    };
   }
 
+  componentWillMount = () => {
+    const { router } = this.props;
+    const activePlatform = ['native', 'web'].find(platform =>
+      router.isActive({ query: { platform } }),
+    );
+    if (activePlatform) {
+      this.state = { selected: activePlatform };
+    } else if (typeof window !== 'undefined') {
+      this.props.router.replace({
+        pathname: window.location.pathname,
+        query: { platform: this.getSelectedSubpage() },
+      });
+    }
+  };
+
+  onClick = selected => {
+    if (typeof window !== 'undefined') {
+      this.props.router.push({
+        pathname: window.location.pathname,
+        query: { platform: selected },
+      });
+    }
+    this.setState({ selected });
+  };
+
+  getSelectedSubpage = () => (this.props.nativeSubpage ? 'native' : 'web');
```

Comment:
nit: no need for the extra parenthesis here

Reply:
Prettier did that 🤷‍♂️